deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+Modules=('api' 'bot' 'storage')
+
+for i in "${Modules[@]}"; do
+  cd $i
+  mvn clean install
```

Comment:
You can call `mvn clean install` once before the loop. It will generate target folder for each module separately.